docs/source/modules/topography.rst

```diff
@@ -0,0 +1,136 @@
+
+As mentioned above, this module is generally not called anymore, in the normal Isca set up, land masks are actually specified through the ``idealized_moist_phys`` module, and the model topography through the ``spectral_init_cond`` module. The main use of it is to provide the "subgrid topography" when using the orographic gravity wave drag scheme (``mg_drag``).
+
+The fields that can be generated with this module are mean and standard deviation of topography within the specified grid boxes; and land-ocean (or water) mask and fractional area. The interpolation conserves the area weighted average of the input data by using the ``horiz_interp`` module.
+
+Namelist options
+^^^^^^^^^^^^^^^^
+| ``topog_file`` - The topography file that you wish to use.
+| ``water_file`` - The water data file (not commonly used, not provided in the Isca release)
+| ``file_in_rad`` - Is the topography file axis measured in radians? (Default False)
+| ``file_in_rad`` - Is the topography file axis measured in degrees? (Default True)
+| ``lat_box_name`` - Name of the latitudinal grid box boundaries of the topog_file. (Default 'latb')
+| ``lon_box_name`` - Name of the longitudinal grid box boundaries of the topog_file. (Default 'lonb')
+
+
+For a typical Earth set up the namelist would simply be:
+
+``'topog_file': ERA_0125.nc``
+
+This essentially just points the ``topography`` module to this file when it is asked for by another subroutine, e.g. ``mg_drag``.
+
+Diagnostics
+^^^^^^^^^^^
+There are no diagnostics available directly through this module. The subgrid topography variance can be obtained through damping driver by asking for ``sgsmtn`` (``mg_drag`` must be turned on).
+
+Relevant subroutines
+^^^^^^^^^^^^^^^^^^^^
+
+NOTE: Some subroutines have dimensional variants, e.g. interp_topog has both a 1d and 2d variant. 
+
+**Public Subroutines**
+
+These subroutines are used by the topography module to produce the land-masks etc that are being asked for by the user. They are largely self explainitory.
+
+| ``get_topog_mean`` returns the mean height from a region of the topography file so that that value can be used as the value when interpolating onto a smaller grid.
+| ``get_topog_stdev`` returns the standard deviation from a region of the topography file so that that value can be assoiated with the same region when interpolating onto a smaller grid.
+| ``get_ocean_frac`` returns the fraction of the land mask that is covered by ocean.
+| ``get_ocean_mask`` returns an ocean/land mask
+| ``get_water_frac`` returns the fraction of the land mask that is covered by water.
+| ``get_water_mask`` returns a water/land mask
```

Comment:
I'm a bit confused by this, sorry: is there a different type of water available in Isca, like lakes/rivers? or is water_frac = ocean_frac?